src/main/java/net/masterthought/cucumber/json/Result.java

```diff
@@ -31,4 +31,9 @@ public String getFormatedDuration() {
     public String getErrorMessage() {
         return StringEscapeUtils.escapeHtml(errorMessage);
     }
+
+    // method used in Velocity templating library
+    public String getErrorMessageTitle() {
+        return errorMessage.split("\\r\\n|\\r|\\n")[0];
```

Comment:
Maybe this one http://stackoverflow.com/questions/454908/split-java-string-by-new-line/454913#454913 ?

and would be nice to move it into final static with friendly name

also what would happen if the first line is very long? how would it be displayed in report?